packages/faustwp-cli/src/telemetry/sendTelemetryData.ts

```diff
@@ -1,39 +1,18 @@
 import 'isomorphic-fetch';
 
+import { getWpUrl } from '../utils/getWpUrl.js';
 import { TelemetryData } from './marshallTelemetryData.js';
-
-const GA_TRACKING_ENDPOINT = 'https://www.google-analytics.com/mp/collect';
-const GA_TRACKING_ID = 'G-KPVSTHK1G4';
-const GA_API_SECRET = '-SLuZb8JTbWkWcT5BD032w';
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const WP_TELEMETRY_ENDPOINT = `${getWpUrl()!}/wp-json/faustwp/v1/process_telemetry`;
 
 /**
- * Send the marshalled telemetry data to GA.
+ * Send the marshalled telemetry data to Faust Telemetry API.
  *
- * @param payload The data being sent to GA
- * @param anonymousId The anonymous ID of the machine we captured during init
+ * @param payload The data being sent to telemetry API
  */
-export const sendTelemetryData = (
-  payload: TelemetryData,
-  anonymousId: string,
-) => {
-  const body = {
-    client_id: anonymousId,
-    events: [
-      {
-        name: 'telemetry_event',
-        params: payload,
-      },
-    ],
-  };
-
-  return fetch(
-    `${GA_TRACKING_ENDPOINT}?${new URLSearchParams({
-      api_secret: GA_API_SECRET,
-      measurement_id: GA_TRACKING_ID,
-    }).toString()}`,
-    {
-      method: 'POST',
-      body: JSON.stringify(body),
-    },
-  );
+export const sendTelemetryData = (payload: TelemetryData) => {
+  return fetch(WP_TELEMETRY_ENDPOINT, {
+    method: 'POST',
```

Comment:
This should only be sent with the `FAUST_SECRET_KEY` as a header:

```suggestion
    method: 'POST',
    headers: {
      'x-faustwp-secret': getWpSecret(),
      'Content-Type': 'application/json'
    }
```

Reply:
Additionally, we should probably only send telemetry data if the WordPress site is over `https` or it's a local site, since we will always be sending this data to WordPress because that is where the telemetry decision is made.

cc @mindctrl